general-configuration-guide/multiple-editors.md

```diff
@@ -25,14 +25,12 @@ The following example breaks the page into two separate editable areas. Each are
 </head>
 
 <body>
-  <h1>Multple editors on a page: Section 1</h1>
   <form method="post">
```

Comment:
I just realised, none of these make sense in a form as they are a div element... Maybe change the divs to a textarea?

Reply:
after discussion, retaining `div`s, added `id`s